Moderation_Matrix.md

```diff
@@ -0,0 +1,62 @@
+# Moderation Matrix
+This matrix correlates with the [Community Rules](https://www.theodinproject.com/community_rules) that are required to be followed in order to participate in the community. Some rules have different levels of severity, so you may see references to them under different numbers of zaps. 
+
+This matrix is not all encompassing and zaps are given at the discretion of the moderator team.
+
+## 0 Zaps (Heads Up)
+- Dog piling/short circuiting a teaching moment
+- Asking about or encouraging the use of Windows
+- "Chat bombing", aka posting many short (ususally 4-5 or fewer words) messages in rapid succession, repeatedly sending parts of thoughts (incomplete, non-standalone sentences or fragments) in new messages. (Use shift + enter and send ONE message where possible or edit your message) 
+- Discussing mental health issues in depth or seeking mental health help from other users
+- Inappropriate usernames and profile pictures (E.g. suggestive or inappropriate memes)
+
+## 1 Zap (WARNING) 
+Minor violations such as:
+- A second infraction of any 0 Zap issues above
+- "Mini-modding"
+  - Please leave moderation action to the moderators
+  - use ModMail to report rule breaking or if you see something you’re unsure about in the community
+- Asking for advice/help on projects unrelated to The Odin Project, or asking about topics unrelated to our Discord channels
+- Mild unprofessionalism:
+  - Drug related conversation 
+  - Treating others on the server disrespectfully
+  - Relationship advice
+
+## 2 Zaps (WARNING)
+Medium severity violations such as: 
+- Suggesting piracy 
+- Unsolicited pings/DM/friend request 
+- Political or religious topics 
+- Mild toxicity:
+  - Directed insults
+  - Provoking/baiting other users
+  - Passive aggression
+  - Naming & shaming
+- Self promo without permission (ask with Modmail)
+  - Do not share resources you have created solely for personal or monetary gain
+  - Examples: "Fill out my survey" (does not help the goals of Odin), "Join my boot camp/program/server" (Could mislead learners), "Why PHP is superior" (unrelated blog post)
+
+## 5 Zaps (SEVERE WARNING)
+High severity violations such as: 
+- Posting about any illegal activities, such as facilitating piracy 
+- Publicly arguing about receiving moderation activity after being asked to take it to ModMail/discuss privately 
+- Excessive toxicity (targeted harassment, inciting drama, aggressively arguing purposefully, etc.) 
+
+## 10 Zaps (Immediate Ban)
```

Comment:
Something I hadn't though of in my previous review, but would this level of zaps result in a *permanent* ban, or would the length of the ban depend on the situation?

This may not matter *too* much if we don't plan on having any other level of zaps result in a temp ban or anything (going off a previous comment from Sully that I will also comment on), but thought it might be worth thinking about/a discussion.

Reply:
It's permanent imo- these are all things that would bar you from participating ever again, and if you're continually breaking rules and rack up this many points, you aren't being a good member of this community.